yarn.lock

```diff
@@ -754,7 +754,7 @@
     "@babel/helper-module-transforms" "^7.20.11"
     "@babel/helper-plugin-utils" "^7.20.2"
 
-"@babel/plugin-transform-modules-commonjs@^7.21.2", "@babel/plugin-transform-modules-commonjs@^7.21.5":
+"@babel/plugin-transform-modules-commonjs@^7.21.5":
```

Comment:
This is not related with this PR, but I've been getting this change every time I was running any frontend script with `make`, and `yarn install` was executed as a side effect.